libraries/AP_Logger/LogStructure.h

```diff
@@ -2508,7 +2508,7 @@ struct PACKED log_Arm_Disarm {
     { LOG_VISUALODOM_MSG, sizeof(log_VisualOdom), \
       "VISO", "Qffffffff", "TimeUS,dt,AngDX,AngDY,AngDZ,PosDX,PosDY,PosDZ,conf", "ssrrrmmm-", "FF000000-" }, \
     { LOG_VISUALPOS_MSG, sizeof(log_VisualPosition), \
-      "VISP", "QQIffffffffb", "TimeUS,RemTimeUS,CTimeMS,PX,PY,PZ,Roll,Pitch,Yaw,PosErr,AngErr,ResetCnt", "sssmmmddhmd-", "FFC00000000-" }, \
+      "VISP", "QQIffffffffb", "TmUS,RemTmUS,CTmMS,PX,PY,PZ,Roll,Pitch,Yaw,PErr,AErr,RstCnt", "sssmmmddhmd-", "FFC00000000-" }, \
```

Comment:
I think we should keep the first column as TimeUS to be consistent with all other messages.  Maybe make the next one RTimeUS and CTimeMS.  worst case let's short Roll, Pitch, Yaw to just R, P, Y.

Reply:
.. and let's keep the column headers consistent between VISP and VISV.  sorry to be annoying.

Reply:
The last commit from this branch seems to work.  If you've got another solution that's fine, i'll stop being pedantic. https://github.com/rmackay9/rmackay9-ardupilot/commits/chobitsfan-ekf3-PosErr

It seems like the limit on the length is 62 or 63 characters.  I guess 64 would be a logical limit.

Reply:
Thank you for helping. I will pull it from your branch

Reply:
It passed all tested, Thank you @rmackay9